### PR TITLE
#7 Fix preview configuration

### PR DESCRIPTION
**Pull request by YokiToki**

### CODE REVIEW

**Review by oyakovlev — APPROVED** [review]

File: storage-service/src/main/kotlin/com/icerockdev/service/storage/preview/PreviewScaleMethod.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.icerockdev.service.storage.preview
```
Comment (Alex009) [inline storage-service/src/main/kotlin/com/icerockdev/service/storage/preview/PreviewScaleMethod.kt:1]: copyright please

Comment (Alex009) [reply storage-service/src/main/kotlin/com/icerockdev/service/storage/preview/PreviewScaleMethod.kt:1]:
```
/*
 * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
 */
```

File: storage-service/src/main/kotlin/com/icerockdev/service/storage/preview/PreviewScaleMethod.kt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import com.sksamuel.scrimage.ScaleMethod
+
+enum class PreviewScaleMethod {
+    FastScale,
+    Lanczos3,
+    BSpline,
+    Bilinear,
+    Bicubic;
+
+    fun getScale(): ScaleMethod {
```
Comment (Alex009) [inline storage-service/src/main/kotlin/com/icerockdev/service/storage/preview/PreviewScaleMethod.kt:12]: internal please. without internal you can get same problem as you fix in `PngPreviewImpl`

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: storage-service/src/main/kotlin/com/icerockdev/service/storage/preview/PreviewScaleMethod.kt (hunk 1 above)
Comment (YokiToki) [reply storage-service/src/main/kotlin/com/icerockdev/service/storage/preview/PreviewScaleMethod.kt:5]: Fixed

File: storage-service/src/main/kotlin/com/icerockdev/service/storage/preview/PreviewScaleMethod.kt (hunk 2 above)
Comment (YokiToki) [reply storage-service/src/main/kotlin/com/icerockdev/service/storage/preview/PreviewScaleMethod.kt]: Fixed

**Review by Alex009 — APPROVED** [review]

**Pull request merged**